runelite-client/src/main/java/net/runelite/client/plugins/achievementdiary/diaries/KourendDiaryRequirement.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2018, Marshall <https://github.com/marshdevs>
+ * Copyright (c) 2018, Adam <Adam@sigterm.info>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.achievementdiary.diaries;
+
+import net.runelite.api.Quest;
+import net.runelite.api.Skill;
+import net.runelite.client.plugins.achievementdiary.GenericDiaryRequirement;
+import net.runelite.client.plugins.achievementdiary.SkillRequirement;
+import net.runelite.client.plugins.achievementdiary.QuestRequirement;
+
+public class KourendDiaryRequirement extends GenericDiaryRequirement
+{
+	public KourendDiaryRequirement()
+	{
+		//EASY
+		add("Mine some Iron at the Mount Karuulm mine.",
+			new SkillRequirement(Skill.MINING, 15));
+		add("Steal from a Hosidius Food Stall.",
+			new SkillRequirement(Skill.THIEVING, 25));
+		add("Browse the Warrens General Store.",
+			new QuestRequirement(Quest.THE_QUEEN_OF_THIEVES, true));
+		add("Enter your Player Owned House from Hosidius.",
+			new SkillRequirement(Skill.CONSTRUCTION, 25));
+		add("Create a Strength potion in the Lovakengi Pub.",
+			new SkillRequirement(Skill.HERBLORE, 12));
+		add("Fish a Trout from the River Molch.",
+			new SkillRequirement(Skill.FISHING, 20));
+
+		//MEDIUM
+		add("Travel to the Fairy Ring south of Mount Karuulm.",
+			new QuestRequirement(Quest.FAIRYTALE_II__CURE_A_QUEEN, true));
+		add("Use Kharedst's memoirs to teleport to all five cities in Great Kourend.",
+			new QuestRequirement(Quest.THE_DEPTHS_OF_DESPAIR, false),
```

Comment:
Adding `false` to the quest requirements is not required and is redundant as it already defaults that way.